CI/windows/02_build_obs.ps1

```diff
@@ -90,7 +89,7 @@ function Configure-OBS {
         "$(if (Test-Path Variable:$Quiet) { "-Wno-deprecated -Wno-dev --log-level=ERROR" })"
     )
 
-    Invoke-Expression "cmake ${CmakeCommand}"
+    Invoke-External cmake -S . -B  ${BuildDirectoryActual} @CmakeCommand
```

Comment:
Should `${BuildDirectoryActual}` be quoted to handle paths with spaces?

Reply:
I'll check this separately, I _think_ Powershell will apply magic quoting here as well, but will amend appropriately if checks prove me otherwise.

Reply:
Tested locally with "build test" as the "BuildDirectory" parameter. While it initially succeeds at creating the directory, CMake breaks once invoked.

Reply:
Build directory string is now quoted.